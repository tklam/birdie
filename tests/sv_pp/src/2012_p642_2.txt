`define wordsize 8
logic [1:`wordsize] data;
//define a nand with variable delay
`define var_nand(dly) nand #(dly)
`var_nand(2) g121 (q21, n10, n11);
`var_nand(5) g122 (q22, n10, n11);
