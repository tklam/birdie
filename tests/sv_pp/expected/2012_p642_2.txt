
logic [1:8] data;
//define a nand with variable delay

nand #(2) g121 (q21, n10, n11);
nand #(5) g122 (q22, n10, n11);
